src/pages/SignupPage.jsx

```diff
@@ -19,7 +23,7 @@ export const SignupPage = () => {
           <Input label="firs name" />
           <Input label="last name" />
         </Grid>
-        <Input label="email" />
+        <Input label="email" className = {classes.szerokosc}/>
```

Comment:
"szerokocs" I don't understand that word :P

Reply:
It was just for try. To delete later xp